tests/unit/amo/components/TestAddonMeta.js

```diff
@@ -53,6 +65,31 @@ describe('<AddonMeta>', () => {
       });
       expect(getUserCount(root)).toMatch(/^1\.000/);
     });
+
+    it('links to stats if add-on author is viewing the page', () => {
+      const addon = {
+        ...fakeAddon,
+        slug: 'coolio',
+        authors: [
+          {
+            id: 11,
```

Comment:
Even though `fakeAddon` is not complete, it's good practice for future maintenance to always extend from the base object, like:

```js
authors: [
  {
    ...fakeAddon.authors[0],
    id: 11,
    name: 'tofumatt',
  },
],
```

The `fakeAddon` author could use some more fields in it too.